src/index.html

```diff
@@ -1,13 +1,80 @@
 <!doctype html>
+
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <title>Product cards</title>
-    <link rel="stylesheet" href="./styles/index.scss">
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+    <link
+      rel="stylesheet"
+      href="./styles/index.scss"
+    >
   </head>
-  <body>
-    <h1>Product cards</h1>
+
+  <body class="page">
```

Comment:
it's redundant to add a class to a body, since we have only one body on a page, so you can just use a tag selector instead
```suggestion
  <body>
```